.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
 node_js:
   - "10"
-  - "9"
   - "8"
   - "6"
-  - "4"
-script: "npm run jshint && npm run test-cover"
+script: "ln -s .. node_modules/sharedb; npm run jshint && npm run test-cover"
```

Comment:
As I mentioned in https://github.com/share/sharedb/pull/227/files#diff-354f30a63fb0907d4ad57269548329e3R8, after thinking over it a bit more, I'm OK with this workaround for now to unblock the failing tests, as the workaround is just isolated to Travis.

Can you add a comment above about this workaround?

Reply:
`ln -s .. node_modules/sharedb` creates a link to `sharedb` in `node_modules`, so that `node` can find it when `sharedb` is loaded by package name in `sharedb-mingo-memory`.

The problem is a circular dependency between `sharedb` and `sharedb-mingo-memory`. `npm install` does not install `sharedb` in `node_modules` because it is redundant considering that the dependencies are installed *for* sharedb, however, `sharedb-mingo-memory` tries to load `sharedb` by package name, so `node` looks for it in `node_modules`.

Reply:
Oh man what a mess. Why does ShareDB depend on `sharedb-mingo-memory` at all? Intuitively it should not.

Reply:
I totally agree with @curran .

Reply:
OK, cool. We'll merge this to get travis working in the meantime and then we can address decoupling subsequently.